README.md

```diff
@@ -8,25 +8,26 @@
 
 In order to make it work you need [Docker](https://docs.docker.com).
 
-Once you've your docker environment ready, clone this repo.
+Once you've your docker environment ready, download the tar file from the disc, un-tar the files, go into the unfetter-insight directory, and compose the flask.
```

Comment:
I'm not sure I understand this.  What is the file from disc?  In this environment, everything is from the perspective of GitHub, so the instructions should only include how to get it running from GitHub/Docker